README.md

```diff
@@ -5,7 +5,7 @@ To view (HTML) files can be [read directly](https://w3c.github.io/rdf-tests/).
 
 The index.html files are built using a Ruby tool-chain based on the corresponding manifest.ttl files. Update using `rake`. Requires Ruby, rdf-turtle and json-ld gems to be installed.
 
-NB: as of October 2025, the test suites for RDF and SPARQL are jointly managed the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
+Currently, the test suites for RDF ([rdf/](rdf/)) and SPARQL ([sparql/](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```

Comment:
```suggestion
Currently, the test suites for RDF ([`rdf/`](rdf/)) and SPARQL ([`sparql/`](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```

Reply:
TallTed, are you sure such relative URLs work as expected? I have my doubts about it (especially when the README is displayed on the repo's landing page)...

Reply:
> are you sure such relative URLs work as expected?

The [relative URLs were inserted](https://github.com/w3c/rdf-tests/pull/224/commits/21be05dfa605cd103fa08b94b29a5807ac2272e3#r2394613415) by @afs.

I just tweaked their visual presentation.